src/app/reviews/reviews.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { ReviewsService, Review } from './reviews.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-reviews',
+  templateUrl: './reviews.component.html',
+  styleUrls: ['./reviews.component.scss']
+})
+export class ReviewsComponent implements OnInit {
+
+  public reviews:Array <Review>;
+  public showDo: boolean = false;
+  public showDone: boolean = true;
```

Comment:
From our logic, display "Do" page and "Done" page can't happen at the same time, so probably we just need one boolean variable for this. Don't need `showDo` and `showDone` both